push-sender/src/main/java/org/jboss/aerogear/unifiedpush/message/sender/fcm/ConfigurableFCMSender.java

```diff
@@ -36,11 +36,15 @@ public ConfigurableFCMSender(String key) {
 
 	@Override
 	protected HttpURLConnection getConnection(String url) throws IOException {
-		
+
 		// let's see if there is a different URL we should post to (e.g. load/stress testing)
 		final String fcmURL = ConfigurationUtils.tryGetProperty(CUSTOM_AEROGEAR_FCM_PUSH_HOST, FCM_ENDPOINT_HOST);
-		
+
 		return (HttpURLConnection) new URL(fcmURL).openConnection();
 	}
-	
+
+	@Override
+	protected HttpURLConnection post(String url, String contentType, String body) throws IOException {
+		return super.post(FCM_ENDPOINT_HOST, contentType, body);
```

Comment:
how about using `ConfigurationUtils.tryGetProperty(CUSTOM_AEROGEAR_FCM_PUSH_HOST, FCM_ENDPOINT_HOST)` instead of just the hard coded value `FCM_ENDPOINT_HOST` ?

Reply:
I prefer not to change the original code.
Furthermore, the getConnection() function is misleading since it's not called in the same flow so I don't want to influence that.

Reply:
ok, though i didn't suggest you'd use `getConnection()` method but rather only to have an ability to overwrite the hard url through configuration

```
@Override
protected HttpURLConnection post(String url, String contentType, String body) throws IOException {
        String fcmUrl = ConfigurationUtils.tryGetProperty(CUSTOM_AEROGEAR_FCM_PUSH_HOST, FCM_ENDPOINT_HOST);
	return super.post(fcmUrl, contentType, body);
}
```